software/firmware/europi.py

```diff
@@ -83,6 +83,20 @@
 HIGH = 1
 LOW = 0
 
+# Pin assignments

```

Comment:
[Discussion] It might be nice to order this section by the pin number, in order to make it easier to find where a specific pin is used. We already have a section ordered by function down where these are used. So then we would have both orders represented.

Reply:
I personally disagree; checking where any given pin is actually used feels like it's the role of the circuit diagrams or hardware documentation. IMO the software should group the definitions in the logical how-it's-used order, not the physical what-is-used order.

Reply:
I believe it was an intentional design decision not to create constants for the pins with the intention that the EuroPi API / Client Library would abstract that away and users wouldn't have any need to reference them. The Diagnostic script is an understandable exception as a testing utility script. That being said, I don't have any objections to making these constants because it cleans up the europi.py code and will help make future work easier.

Reply:
Yeah, I don't expect these constants to be used in much besides calibration & testing scripts. The main goal was just a code cleanup.